scripts/run_portserver.py

```diff
@@ -419,21 +419,21 @@ def __init__(self, handler, socket_path):
         self.socket = self._start_server(self.socket_path)
         self.handler = handler
 
-    def run(self):
+    def run(self) -> None:
         """Run the server in an infinite loop.
 
         Spawns a thread to handle each connection to the socket. Uses
         the handle_connection function to handle each connection.
         """
-        while True:
+        while True: # pragma: no cover
```

Comment:
We exit from this infinite loop using the `KeyboardInterrupt (by pressing ^C)` which simply displays info and exits via `sys.exit()`. But while testing the code, I cannot find a way to mock the interrupt since `sys.exit()` would totally stop the execution rather than simply breaking away from the loop.

Reply:
OK, that makes sense. Maybe use `# pragma: no branch` instead and leave an explanatory comment?

Reply:
`pragma: no branch` would not work here since we are not moving inside the infinite loop and hence it will show uncovered lines too. I have added an explanatory comment regarding the same now.